packages/vue-components/src/Navbar.vue

```diff
@@ -52,11 +60,17 @@ export default {
       default: 'sibling-or-child',
     },
   },
+  provide() {
+    return {
+      toggleLowerNavbar: this.toggleLowerNavbar,
```

Comment:
how about just a prop? since they're in the same compilation scope

Reply:
Hmm just to clarify, if this is passed to `<page/site-nav-button>` component through a prop, since `lower-navbar` is a slot, if a user defines new content in the layout file for this slot without specifying the prop, for example,
```
<div slot="lower-navbar">
  <site-nav-button />
</div>
``` 
then it won't be passed down to the component in this case?

Reply:
https://github.com/MarkBind/markbind/pull/1445#discussion_r571418385